Sources/Basics/Triple.swift

```diff
@@ -10,173 +10,36 @@
 //
 //===----------------------------------------------------------------------===//
 
-import protocol Foundation.CustomNSError
-import var Foundation.NSLocalizedDescriptionKey
-
 import enum TSCBasic.JSON
 import class TSCBasic.Process
+import struct SwiftDriver.Triple
```

Comment:
This file should probably be renamed to Triple+Extension to match style of repo

Reply:
Fixed!